components/schema-migrator/migrations/201908141300_initialize_schema.up.sql

```diff
@@ -0,0 +1,207 @@
+-- Runtime
+
+CREATE TYPE runtime_status_condition AS ENUM (
+    'INITIAL',
+    'READY',
+    'FAILED'
+);
+
+CREATE TABLE runtimes (
+    id uuid PRIMARY KEY,
+    tenant_id uuid NOT NULL,
+    name varchar(256) NOT NULL,
+    description text,
+    status_condition runtime_status_condition DEFAULT 'INITIAL' ::runtime_status_condition NOT NULL,
+    status_timestamp timestamp NOT NULL,
+    auth jsonb
+);
+
+ALTER TABLE runtimes
+    ADD CONSTRAINT runtime_id_name_unique UNIQUE (tenant_id, name);
+
+-- Application
+
+CREATE TYPE application_status_condition AS ENUM (
+    'INITIAL',
+    'UNKNOWN',
+    'READY',
+    'FAILED'
+);
+
+CREATE TABLE applications (
+    id uuid PRIMARY KEY,
+    tenant_id uuid NOT NULL,
+    name varchar(36) NOT NULL,
```

Comment:
I've limited it because of:
https://github.com/kyma-incubator/compass/issues/213